exercises/practice/pov/pov.go

```diff
@@ -1,23 +1,52 @@
 package pov
 
-// Define the Graph type here.
+// Define the Tree type here
 
-func New() *Graph {
-	panic("Please implement the New function")
+type Tree struct{}
+
+// New creates and returns a new Tree with the given root value and children
```

Comment:
All comments that are full sentences should end with a period. https://github.com/golang/go/wiki/Comments#comments

Reply:
Done, I think.